README.md

```diff
@@ -16,82 +20,811 @@ yarn add gotrue-js
+  Set-Cookie: PHPSESSID=77c421bf85fa412e5f994f28a6b30956; expires=Wed, 16-May-2018 07:19:13 GMT; path=/; secure; HttpOnly
+  Expires: Thu, 19 Nov 1981 08:52:00 GMT
+  Cache-Control: no-store, no-cache, must-revalidate, post-check=0, pre-check=0
+  Pragma: no-cache
+  Set-Cookie: PHPSESSID=77c421bf85fa412e5f994f28a6b30956; expires=Wed, 16-May-2018 07:19:13 GMT; path=/; secure; httponly
+  Location: https://example.netlify.com/#confirmation_token=Iyo9xHvsGVbW-9A9v4sDmQ
+```
+
+Example usage:
+
+```js
+auth
+  .confirm(token)
+  .then(function(response) {
+    console.log("Confirmation email sent", JSON.stringify({ response }));
+  })
+  .catch(function(e) {
+    console.log(e);
+  });
+```
+
+_This method requires usage of browser window object `localStorage`. Test the usage in your front end code._
+
+Example response object:
+
+```js
+{
+  "response": {
+    "api": {
+      "apiURL": "https://example.netlify.com/.netlify/identity",
+      "_sameOrigin": true,
+      "defaultHeaders": {}
+    },
+    "url": "https://example.netlify.com/.netlify/identity",
+    "token": {
+      "access_token": "example-jwt-token",
+      "token_type": "bearer",
+      "expires_in": 3600,
+      "refresh_token": "example-refresh_token",
+      "expires_at": 1526110512000
+    },
+    "id": "example-id",
+    "aud": "",
+    "role": "",
+    "email": "example@netlify.com",
+    "confirmed_at": "2018-05-12T06:35:13Z",
+    "confirmation_sent_at": "2018-05-12T06:34:35Z",
+    "app_metadata": {
+      "provider": "email"
+    },
+    "user_metadata": {},
+    "created_at": "2018-05-12T06:34:35Z",
+    "updated_at": "2018-05-12T06:34:35Z"
+  }
+}
+```
+
+### Login a user
+
+Handles user login via the specified email and password
+
+`auth.login(email, password)`
+
+Example usage:
+
+```js
+auth
+  .login(email.value, password.value)
+  .then(response => {
+    showMessage("Success! Response: " + JSON.stringify({ response }), form);
+  })
+  .catch(error => showMessage("Failed :( " + JSON.stringify(error), form));
+```
+
+Example response object:
+
+```js
+{
+  "response": {
+    "api": {
+      "apiURL": "https://example.netlify.com/.netlify/identity",
+      "_sameOrigin": true,
+      "defaultHeaders": {}
+    },
+    "url": "https://example.netlify.com/.netlify/identity",
+    "token": {
+      "access_token": "example-jwt-token",
+      "token_type": "bearer",
+      "expires_in": 3600,
+      "refresh_token": "example-refresh_token",
+      "expires_at": 1526062471000
+    },
+    "id": "example-id",
+    "aud": "",
+    "role": "",
+    "email": "example@netlify.com",
+    "confirmed_at": "2018-05-04T23:57:17Z",
+    "app_metadata": {
+      "provider": "email"
+    },
+    "user_metadata": {},
+    "created_at": "2018-05-04T23:57:17Z",
+    "updated_at": "2018-05-04T23:57:17Z"
+  }
+}
+```
+
+### Request password recover email
+
+This function sends a request to GoTrue API and triggers a password recovery email to the specified email address
+
```

Comment:
We should again mention, the token comes in on a hash that will be linked to from an email.